.clang-format

```diff
@@ -260,6 +260,23 @@ StatementMacros:
   - _END_EXTERN_C
   - _EXTERN_C_UNLESS_PURE
   - _END_EXTERN_C_UNLESS_PURE
+  - _NODISCARD_TRY_CHANGE_STATE

```

Comment:
```suggestion
  - _NODISCARD_TRY_CHANGE_STATE
  - _NODISCARD_CTOR_LOCK
```

Reply:
If you add `_NODISCARD_CTOR_LOCK`. then more lines are affected.

Reply:
Updated